src/searchee.ts

```diff
@@ -100,20 +108,55 @@ export async function createSearcheeFromTorrentFile(
 }
 
 export async function createSearcheeFromPath(
-	filepath: string,
+	root: string,
 ): Promise<Result<Searchee, Error>> {
-	const files = getFilesFromDataRoot(filepath);
+	const files = getFilesFromDataRoot(root);
 	if (files.length === 0) {
-		return resultOfErr(new Error("No files found"));
+		const msg = `No files found in ${root}`;
+		logger.verbose({
+			label: Label.PREFILTER,
+			message: msg,
+		});
+		return resultOfErr(new Error(msg));
 	}
 	const totalLength = files.reduce<number>(
 		(runningTotal, file) => runningTotal + file.length,
 		0,
 	);
+
+	const baseName = basename(root);
+	const seasonMatch =
+		baseName.length < 12 ? baseName.match(SONARR_SUBFOLDERS_REGEX) : null;
+	if (!seasonMatch) {
+		return resultOf({
+			files: files,
+			path: root,
+			name: baseName,
+			length: totalLength,
+		});
+	}
+	// Title is just SXX or Season XX, need to find title above or below
```

Comment:
is there value in going in both directions rather than just looking at the direct parent?

Reply:
Checking from the `EP_REGEX` is preferred as the title is highly unlikely to be junk. The check above isn't necessary but might help if users don't have their sonarr episodes named correctly or bad torrent data naming.